test/utils/configuration/configuration.go

```diff
@@ -28,6 +29,7 @@ import (
 const (
 	envVarMCPUpdateTimeout  = "NROP_E2E_MCP_UPDATE_TIMEOUT"
 	envVarMCPUpdateInterval = "NROP_E2E_MCP_UPDATE_INTERVAL"
+	envVarPlatform          = "NROP_E2E_PLATFORM"
```

Comment:
Since the tests are going to be consumed by external projects, are all these environment variables are documented somewhere?
We can put them [here](https://github.com/openshift-kni/numaresources-operator/blob/main/test/e2e/serial/README.md) for start